### PR TITLE
fix Schema types and operation imports naming

### LINKED ISSUE #108: Harmonize ordering and naming of Schema Types and Function/Action Imports wizard pages
### Issue description
1. Currently, if say I have a function import _GetNearestAirport_ as the only object that depends on _Airport_ entity type, and I don't need either to be emitted on the Connected Service proxy, I'm forced to first skip **Schema Types** wizard page, go to the **Function/Action Imports** wizard page and uncheck it, before going back to the **Schema Types** wizard page to uncheck _Airport_
2. The title on the **Schema Types** wizard page is **Schema Selection** while the title on the **Function/Action Imports** wizard page is **Object Selection**

#### Proposed solution
1. Since **Function/Action Imports** depends on **Schema Types** and not the other way around, I believe the logical thing would be to have **Function/Action Imports** wizard page appear above **Schema Types** wizard page. That way, if I uncheck a function or action import on the **Function/Action Imports** wizard page, then as I proceed to the **Schema Types** wizard page I'll not having any trouble unchecking the schema types it depends on in a scenario similar to the one described above
2. Harmonize the naming:-
   - Title for **Schema Types** wizard page could be **Schema Types Selection**
   - Title for **Function/Action Imports** wizard page could be **Function/Action Imports Selection**
   - Or any suitable naming so long as its harmonious

#### Describe alternatives you've considered
N/A

#### Additional context
N/A

### PR DESCRIPTION
**Pull request by paulodero**
Fixes #108

### CODE REVIEW

File: src/packages.config [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="EnvDTE" version="8.0.2" targetFramework="net472" />
-  <package id="Microsoft.OData.Client" version="7.6.3" targetFramework="net472" />
```
Comment (marabooy) [inline src/packages.config:4]: Not sure these are needed

File: test/ODataConnectedService.Tests/ODataConnectedService.Tests.csproj [hunk 2]
```diff
@@ -76,7 +76,7 @@
       <Version>2.0.0.1</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.OData.Client">
-      <Version>7.6.3</Version>
```
Comment (marabooy) [inline test/ODataConnectedService.Tests/ODataConnectedService.Tests.csproj:79]: Same here

File: src/packages.config (hunk 1 above)
Comment (paulodero) [reply src/packages.config:4]: Is it a problem to reference the latest OData Client?

Comment (marabooy) [reply src/packages.config:4]: I would probably keep it at a minimum version. been seeing people having some regressions with this release. two new bugs got added on odata.net so we may need to keep the old version. I know currently we havent upgraded the current referenced libs in the build. Maybe we should have one version both in t4 and in this.

Comment (paulodero) [reply src/packages.config]: Reverted.

File: test/ODataConnectedService.Tests/ODataConnectedService.Tests.csproj (hunk 2 above)
Comment (paulodero) [reply test/ODataConnectedService.Tests/ODataConnectedService.Tests.csproj]: Reverted.

**Pull request merged**